test/utils_test.go

```diff
@@ -99,11 +99,13 @@ func createRouters(t *testing.T, num int, batcherInfos []nodeconfig.BatcherInfo,
 		bundle.ConfigtxValidatorReturns(configtxValidator)
 
 		config := &nodeconfig.RouterNodeConfig{
-			ListenAddress:      "0.0.0.0:0",
-			ConfigStorePath:    t.TempDir(),
-			TLSPrivateKeyFile:  kp.Key,
-			TLSCertificateFile: kp.Cert,
-			PartyID:            types.PartyID(i + 1),
+			ListenAddress:           "0.0.0.0:0",
+			MonitoringListenAddress: "0.0.0.0:0",
```

Comment:
It is much better/safer to bind to `127.0.0.1` in tests, 0.0.0.0 is bind all, which is listening to all external comm